scripts/o3de/ExportScripts/export_utility.py

```diff
@@ -0,0 +1,173 @@
+                         engine_centric=engine_centric,
+                         fail_on_ap_errors=fail_on_asset_errors,
+                         using_installer_sdk=is_installer_sdk,
+                         tool_config=tool_config,
+                         selected_platform=ap_selected_platform,
+                         logger=logger)
+    
+    expected_bundles_path = exp.bundle_assets(ctx=ctx,
+                      selected_platform=ab_selected_platform,
+                      seedlist_paths=seedlist_paths,
+                      seedfile_paths=seedfile_paths,
+                      tools_build_path=tools_build_path,
+                      engine_centric=engine_centric,
+                      asset_bundling_path=asset_bundling_path,
+                      using_installer_sdk=is_installer_sdk,
+                      tool_config=tool_config,
+                      max_bundle_size=max_bundle_size)
+
+    return expected_bundles_path
+
+
+def create_common_export_params_and_config(parser, export_config):
+    parser.add_argument(exp.CUSTOM_SCRIPT_HELP_ARGUMENT,default=False,action='store_true',help='Show this help message and exit.')
+    
+    default_project_build_config = export_config.get_value(key=exp.SETTINGS_PROJECT_BUILD_CONFIG.key,
+                                                            default=exp.SETTINGS_PROJECT_BUILD_CONFIG.default)
+    parser.add_argument('-cfg', '--config', type=str, default=default_project_build_config, choices=[exp.BUILD_CONFIG_RELEASE, exp.BUILD_CONFIG_PROFILE, exp.BUILD_CONFIG_DEBUG],
+                        help='The CMake build configuration to use when building project binaries.')
+    
+
+    default_tool_build_config = export_config.get_value(key=exp.SETTINGS_TOOL_BUILD_CONFIG.key,
+                                                            default=exp.SETTINGS_TOOL_BUILD_CONFIG.default)
+    parser.add_argument('-tcfg', '--tool-config', type=str, default=default_tool_build_config, choices=[exp.BUILD_CONFIG_RELEASE, exp.BUILD_CONFIG_PROFILE, exp.BUILD_CONFIG_DEBUG],
+                        help='The CMake build configuration to use when building tool binaries.')
+
+    default_tools_build_path = export_config.get_value(exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.key, exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.default)
+    parser.add_argument('-tbp', '--tools-build-path', type=pathlib.Path, default=pathlib.Path(default_tools_build_path),
+                        help=f"Designates where the build files for the O3DE toolchain are generated. If not specified, default is '{default_tools_build_path}'.")
+
+    default_max_size = export_config.get_value(exp.SETTINGS_MAX_BUNDLE_SIZE.key, exp.SETTINGS_MAX_BUNDLE_SIZE.default)
+    parser.add_argument('-maxsize', '--max-bundle-size', type=int, default=int(default_max_size),
+                        help=f"Specify the maximum size of a given asset bundle.. If not specified, default is {default_max_size}.")
```

Comment:
Having a `default` implies a value that is set when not specified, no need to spell that out.

```suggestion
                        help=f"Specify the maximum size of a given asset bundle in MB. 
                        (default {default_max_size}).")
```